docs/developerguide.rst

```diff
@@ -2,3 +2,40 @@
 
 Developer guide
 ===============
+
+This tutorial explains how to quick-start with REANA-UI.
```

Comment:
Nice, super simple guide! What do you think about simplifying even a bit more? Merging this line with the line after the subsection `Deploy locally` to something like: `In this guide we will setup a development instance of the REANA-UI with live reloading.` and then start directly with the list of steps to follow, so we remove first subsection since we will not have a new one soon? Otherwise looks great to me!

Reply:
Done ✅